test/api_test.py

```diff
@@ -232,3 +233,52 @@ def test_send_to_device(self):
         req = responses.calls[0].request
         assert req.url == send_to_device_url
         assert req.method == 'PUT'
+
+
+class TestRoomApi:
+    cli = client.MatrixClient("http://example.com")
+    user_id = "@user:matrix.org"
+    room_id = "#foo:matrix.org"
+
+    @responses.activate
+    def test_create_room(self):
```

Comment:
I'd argue we should be splitting up tests if you are testing different behaviours. E.g. ``test_create_room_federate``, ``test_create_room_visibility``